app/services/users.js

```diff
@@ -0,0 +1,17 @@
+const { databaseError, defaultError } = require('../errors');
+const userModel = require('../models').User;
+
+exports.signUp = user =>
+  userModel
```

Comment:
the method to create is of the model, therefore it would be good to leave it in the model and, from the service, send the object to create. So it is easier to handle errors